pages/implementations/testcases.md

```diff
@@ -2,28 +2,56 @@
 title: Test Cases
 ---
 
-The authored rules, comes with several test cases, which are designed for easy consumption by test tools. The test cases are made available regularly & automatically on every successful commit (to the master branch).
+All ACT rules include a number of test cases, which are designed for easy consumption by accessibility test tools and test methodologies. Test cases are updated regularly as part of rule writing. All test cases are described in a JSON file:
 
  <a class='btn' href='/testcases.json'>
-  See All Test Cases
+  See the Test Cases JSON
  </a>
 
-A test case for a given rule is represented as shown by the below excerpt, where:
+## Test Case Format
 
-- `url`: is the standalone page containing the test case.
-- `expected`: is the expected outcome of the test case, pertaining to the rule.
-- `ruleId`: is the unique identifier for the rule.
-- `rulePage`: is the page containing a detailed rule description.
+In the `testcases.json` file, test cases are included on the `testcases` array, each with the following properties:
+
+- `testcaseId`: Hash of the test case, changes when the test case is updated
+- `url`: Standalone page containing the test case.
+- `expected`: Expected outcome of the test case (`passed`, `failed`, or `inapplicable`)
+- `ruleId`: Unique identifier for the rule.
+- `ruleName`: Title of the rule the test case is for
+- `rulePage`: Page containing a detailed rule description.
+- `requirementsMapping`: Array of success criteria and other accessibility requirements not satisfied when the rule `failed`
 
 ```json
- "a11y-testcases": [
+{
+  "name": "ACT-Rules Community test cases",
+  "license": "https://act-rules.github.io/pages/license/",
+  "count": 558,
+  "testcases": [
     {
-      "url": "//auto-wcag.github.io/auto-wcag/testcases/937e33-passed-example-1.html",
+      "testcaseId": "55f3ed0ec0f324514a0d223b737bc1e4c81593c7",
+      "url": "https://act-rules.github.io/testcases/5f99a7/55f3ed0ec0f324514a0d223b737bc1e4c81593c7.html",
       "expected": "passed",
-      "ruleId": "937e33",
+      "ruleId": "5f99a7",
       "ruleName": "ARIA attribute is valid",
-      "rulePage": "//auto-wcag.github.io/auto-wcag/rules/937e33"
-    },
-    ...
- ]
+      "rulePage": "https://act-rules.github.io/rules/5f99a7",
+      "requirementsMapping": ["wcag20:4.1.2"]
+    }, ... ]
+}
 ```
+
+## Running The Test Cases
+
+To run the test cases, go through each URL run the necessary tests on that page. Tools and test procedures do not need to have a one-to-one mapping to ACT-R rules. While you could run all tests in an implementation on every test case, for the sake of efficiency, we recommend doing one of the following:
```

Comment:
... `one-to-one mapping to ACT Rules` ...

ACT-R rules does not read right...

Reply:
Rules from the ACT-R community group are ACT-R rules.